bot/converters.py

```diff
@@ -382,8 +382,8 @@ async def convert(self, ctx: Context, duration: str) -> datetime:
 class OffTopicName(Converter):
     """A converter that ensures an added off-topic name is valid."""
 
-    ALLOWED_CHARACTERS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ!?'`-<>"
-    TRANSLATED_CHARACTERS = "𝖠𝖡𝖢𝖣𝖤𝖥𝖦𝖧𝖨𝖩𝖪𝖫𝖬𝖭𝖮𝖯𝖰𝖱𝖲𝖳𝖴𝖵𝖶𝖷𝖸𝖹ǃ？’’-＜＞"
+    ALLOWED_CHARACTERS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ!?'`-<>⧹⧸"
```

Comment:
`ALLOWED CHARACTERS` should have the regular forward/backward slashes, no? Unless I'm going mad, what you have now are the SOLIDUSses

Reply:
Oh, right. I made the functionality actually use the unicode characters and tested using them as well. Be back in a minute

Reply:
Tested this time with actual regular slashes and pushed changes 
<img width="403" alt="image" src="https://user-images.githubusercontent.com/75038675/164579612-f65396c3-8dd0-4a4d-82f8-ed51446432fb.png">